src/content/admin/okteto-insights.mdx

```diff
@@ -110,7 +110,7 @@ Okteto will progressively make more data available through this endpoint. The fo
 
 ### Build Metrics
 
-- `okteto_usage_build_duration_seconds`: This metric is a [histogram](https://prometheus.io/docs/practices/histograms/) which represent metrics related to build times of the images that are being built within Okteto. It has the following labels:
+- `okteto_usage_build_duration_seconds`: This metric is a [histogram](https://prometheus.io/docs/practices/histograms/) which represent metrics related to build times of the images that are being built within Okteto. Currently this will only show builds that have successfully completed. It has the following labels:
```

Comment:
Should we also include this change to `1.16` versioned docs?

Reply:
Sorry I missed your comment. This update includes a change for the 1.16 versioned docs